site-content/source/modules/ROOT/pages/blog/Liquibase-is-Ready-to-Support-Cassandra-4.0-Users.adoc

```diff
@@ -0,0 +1,18 @@
+= Liquibase is Ready to Support Cassandra 4.0 Users
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: April 28, 2022
+:page-post-author: The Apache Cassandra Community
+:description: The Apache Cassandra Community
```

Comment:
Page description should be a 5-word (7 at the most) description of the document, similar to the title but more descriptive and shorter:

```suggestion
:description: Liquibase supports Apache Cassandra 4.0
```